src/board_controller/brainalive/brainalive.cpp

```diff
@@ -302,15 +305,20 @@ int BrainAlive::config_board (std::string config)
     {
         return (int)BrainFlowExitCodes::BOARD_NOT_CREATED_ERROR;
     }
-    uint8_t command[5];
-    command[0] = 0x0a;
-    command[1] = 0x81; // it is hardcoded for now only
-    command[2] = 0x00;
-    command[3] = 0x00;
-    command[4] = 0x0d;
-    safe_logger (spdlog::level::trace, config[2]);
+    // Calculate the number of bytes
+    size_t numBytes = config.length () / 2; // Each byte is represented by 2 hex characters
```

Comment:
There is snake case with _ for variables everywhere in brainflow, lets keep it consistent

Reply:
Applies to all variables in this block

Reply:
done